### ISSUE TITLE
get scipy logistic regression working

### ISSUE THREAD

**Issue by josh-marsh** [MatthewJA/thursday#14]
Make images smaller. Use skimage transform.

**Comment by josh-marsh** [comment]
the accuracy is quite horrendous and there are some errors I can't figure out. See Commit ebb4bb2

**Comment by MatthewJA** [comment]
Hard to say why the accuracy is low but maybe this is because of the scaling? You can test this by comparing it to keras/sklearn with scaled inputs.

But the errors look like precision loss, so maybe your numbers are getting too big. I'll have a proper look tomorrow.

**Comment by MatthewJA** [comment]
I can get the accuracy to 57% by rescaling your inputs and fiddling with the loss function so there aren't any nans.

**Comment by MatthewJA** [comment]
Do [this](https://nbviewer.ipython.org/url/machlearn.gitlab.io/isml2017/tutorial/03-classification-work.ipynb) notebook, minus Fisher's discriminant.

**Comment by josh-marsh** [comment]
I know we talked about this, but why were there nans again?

**Comment by MatthewJA** [comment]
log(0)
